Not Found: /report-bug/vvv
"GET /report-bug/vvv HTTP/1.1" 404 7843
Not Found: /uju
"GET /uju HTTP/1.1" 404 8129
Not Found: /uju
"GET /uju HTTP/1.1" 404 8129
Internal Server Error: /trigger-error/
Traceback (most recent call last):
  File "/Users/arnavshah/Documents/mafia3/.venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/arnavshah/Documents/mafia3/.venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/arnavshah/Documents/mafia3/mafiarolesite/myapp/views.py", line 316, in trigger_error
    1/0
    ~^~
ZeroDivisionError: division by zero
"GET /trigger-error/ HTTP/1.1" 500 7447
You're accessing the development server over HTTPS, but it only supports HTTP.
You're accessing the development server over HTTPS, but it only supports HTTP.
You're accessing the development server over HTTPS, but it only supports HTTP.
You're accessing the development server over HTTPS, but it only supports HTTP.
Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/Users/arnavshah/Documents/mafia3/.venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/arnavshah/Documents/mafia3/.venv/lib/python3.12/site-packages/django/utils/deprecation.py", line 128, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/arnavshah/Documents/mafia3/.venv/lib/python3.12/site-packages/django/middleware/security.py", line 28, in process_request
    host = self.redirect_host or request.get_host()
                                 ^^^^^^^^^^^^^^^^^^
  File "/Users/arnavshah/Documents/mafia3/.venv/lib/python3.12/site-packages/django/http/request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Bad Request: /room/6LJVF8/status/
"GET /room/6LJVF8/status/ HTTP/1.1" 400 143
You're accessing the development server over HTTPS, but it only supports HTTP.
You're accessing the development server over HTTPS, but it only supports HTTP.
You're accessing the development server over HTTPS, but it only supports HTTP.
You're accessing the development server over HTTPS, but it only supports HTTP.
